user_guide/intro_log_in_and_edit_profile.rst

```diff
@@ -0,0 +1,554 @@
+
+
+Log In Using Other Corporate Credentials 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+OroCRM can be set up to accept your existing corporate login and password. This is possible if your organization 
+utilizes a common server to share them across multiple applications. This is referred to as LDAP integration. Please 
+check with your system administrator to see if this is the case.
+
+
+.. _user-guide-getting-started-my-user:
+
+Access Your User Page
+---------------------
+
+You can always find a link to your user page in the drop-down menu that resides below your username in the top right 
+corner. This is a fast way to access your user profile, calendar, mailbox, and task list. 
+
+.. image:: ./img/intro/user_menu.png
+
+The following subsection is devoted to the user page that is available with *"My User"* menu. You can find more 
+information about other menu items in the :ref:`User Menu <user-guide-getting-started-user-menu>` document.
+
+.. _user-guide-getting-started-my-user-overview:
+
+My User Page Overview
+^^^^^^^^^^^^^^^^^^^^^
+
+This page provides you the full details of your user profile.
+
+Its *"General"* section contains the basic profile details.
+
+.. image:: ./img/intro/user_view_general.png
+
+The set of information available in the section depends on the 
+:ref:`permissions <user-guide-user-management-permissions-basic>` defined for your user. The information may include:
+
+* Your name and system information such as status (Active), login count, date and time of the last login - in the header
+
+* Your username 
+
+* Your birthday
+
+* Information about your role and place in the company structure, such as your roles in the system and business units 
+  you have access to, your API key (if any)
+  
+* Any additional details of a user defined for your organization (in the example above this is the *"Title"* field).
+
+* A link to Outlook Add-in download (for enterprise users only)
+
+* Tags assigned to your user
+
+If there is an email saved or/and a call logged in OroCRM related to you, they will be displayed in the 
+*"Activities"* section. 
+
+.. image:: ./img/intro/user_view_activities.png
+
+If there is a :ref:`task <user-guide-activities-tasks>` or a :ref:`case <user-guide-activities-cases>` assigned to you, 
+they will be displayed in corresponding grids of the "*Additional"* section.
+
+.. image:: ./img/intro/user_view_additional.png 
+
+
+Actions Available from the "My User" Page
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use Action Buttons
+""""""""""""""""""
+
+Action buttons are available in the top left corner of *"My User"* page. The set of buttons depends on your organization 
+settings and user permissions. These are the same actions that you can perform in the system with regard to other users.
+Most of the actions is described in the :ref:`Productivity Tools section <user-guide-productivity-tools>`.
```

Comment:
Edit: Action buttons are available in the top left corner of the *"My User"* page. The set of buttons depends on your organization settings and user permissions. These are the same actions that you can perform in the system with regard to other users. Most of the actions are described in the :ref:`Productivity Tools section <user-guide-productivity-tools>`.